spec/features/lifehacks/02_user_views_details_of_lifehack_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+require 'spec_helper'
+require 'orderly'
+
+# - [] I should be able to click on a Lifehack list item on the index page to be brought to the show page for that individual Lifehack
+# - [] I should be able to see all details about the lifehack
+# - [] I should be able to see all reviews of the lifehack
+# - [] I should be able to sort reviews by votes and date
+# - [] I should be able to click a link to navigate back to the index page
+
+feature "user sees the details of a lifehack" do
+  scenario "sees the details" do
+
+    first_lifehack = FactoryGirl.create(:lifehack)
+    second_lifehack = FactoryGirl.create(:lifehack)
+
+    visit lifehacks_path
+
+    expect(page).to have_content("#{first_lifehack.created_at.strftime("%m/%d/%Y")} #{first_lifehack.created_at.strftime("%I:%M%p")}")
```

Comment:
Line is too long. [134/80]<br>Prefer single-quoted strings inside interpolations.